packages/flutter_tools/lib/src/ios/ios_deploy.dart

```diff
@@ -87,11 +85,11 @@ class IOSDeploy {
   ///
   /// Uses ios-deploy and returns the exit code.
   Future<int> installApp({
-    @required String deviceId,
-    @required String bundlePath,
-    @required Directory appDeltaDirectory,
-    @required List<String>launchArguments,
-    @required IOSDeviceConnectionInterface interfaceType,
+    required String deviceId,
+    required String bundlePath,
+    required List<String>launchArguments,
+    required IOSDeviceConnectionInterface interfaceType,
+    Directory? appDeltaDirectory,
```

Comment:
is this not always passed? or is it always passed, but sometimes null?

Reply:
Always passed, can be null.
https://github.com/flutter/flutter/blob/e028d0f046c20c61b7c7c8a23bc456f337410658/packages/flutter_tools/lib/src/ios/application_package.dart#L104